string name
string status
float32 cpu
float32 memory_usage
float32 memory_limit
string restart_policy
int32 restart_max_retry
